test/e2e/flux_test.go

```diff
@@ -164,6 +165,57 @@ func TestVSphereKubernetes122GitopsOptionsFlux(t *testing.T) {
 			api.WithFluxNamespace(fluxUserProvidedNamespace),
 			api.WithFluxConfigurationPath(fluxUserProvidedPath),
 		),
+	)
+	runFluxFlow(test)
+}
+
+func TestCloudStackKubernetes120Flux(t *testing.T) {
```

Comment:
We probably don't need all the CloudStack tests here, we only need the ones that would specifically test Cloudstack functionality compared to our tests, so based on this it seems like we only really need one of the Flux tests.

Reply:
Which one of these do you feel is best to leave in that case?